api-spec.json

```diff
@@ -0,0 +1,15640 @@
+{
+   "oappy": "0.0.1",
+   "info": {
+      "name": "enigma",
+      "go-package-name": "enigma",
+      "go-package-import-path": "github.com/qlik-oss/enigma-go",
+      "version": "devbuild",
+      "description": "enigma-go is a library that helps you communicate with a Qlik Associative Engine.",
+      "license": "MIT",
+      "x-qlik-visibility": "public",
+      "x-qlik-stability": "locked"
```

Comment:
For other spec-types we inherit downwards in the tree, i would recommend having this one level up on the root level and not inside the info object